lib/hangman_rules.rb

```diff
@@ -0,0 +1,43 @@
+require_relative 'words'
+
+class HangmanRules
+
+  attr_reader :words, :available_letters, :right_guesses
+
+  def initialize(words)
+  @words = words.get_words()
+  @available_letters = ("a".."z").to_a
+  @right_guesses = []
+  end
+
+  def random_word()
+    words.sample()
+  end
+
+  def keep_track_of_guesses(secret_word, guessed_letter)
+    add_if_right_guess(secret_word, guessed_letter)
+    remove(guessed_letter)
+  end
+
+  def get_letters()
```

Comment:
In Ruby we rarely name things "get" and you already have a way to access the `available_letters` with the `attr_reader`

Reply:
Gotcha